Resources/views/CRUD/base_list.html.twig

```diff
@@ -257,10 +257,10 @@ file that was distributed with this source code.
     {% if admin.datagrid.filters %}
         {% form_theme form admin.getTemplate('filter') %}
 
-        <div class="col-xs-12 col-md-12 sonata-filters-box" style="display: {{ admin.datagrid.hasDisplayableFilters ? 'block' : 'none' }}" id="filter-container-{{ admin.uniqid() }}">
+        <div class="col-xs-12 col-md-12 sonata-filters-box" style="display: {{ admin.datagrid.hasDisplayableFilters and not (admin.isChild and 1 == admin.datagrid.filters|length) ? 'block' : 'none' }}" id="filter-container-{{ admin.uniqid() }}">
```

Comment:
Please linebreak this

Reply:
Also, doesn't bootstrap have a built-in class to hide things? would be better to avoid using `style`

Reply:
Yes, Bootstrap has built-in classes for hiding content https://getbootstrap.com/docs/3.3/css/#helper-classes-show-hide

Reply:
Thanks @TorbenLundsgaard !

Reply:
We need to change the JS part too, if we change this to `class="hidden"`